### PR TITLE
Add: some features & fixes

### PR DESCRIPTION
**Pull request by BeBlood**
Fix some form types (parent & names)
Add GoNextStepEventAction
Add Breadcrumb

### CODE REVIEW

File: Breadcrumb/Breadcrumb.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        foreach ($this->items as $name => $stepNames) {
+            if (in_array($currentStepName, $stepNames)) {
+                return $name;
+            }
+        }
+
+        return null;
+    }
+
+    public function getGoBackStepName()
```
Comment (konandrum) [inline Breadcrumb/Breadcrumb.php]: missing return type :string

File: Breadcrumb/Breadcrumb.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            return $takenPathSource;
+        }
+
+        throw new InvalidDestinationException(
+            $this->navigator->getCurrentStep()->getName(),
+            sprintf('BreadcrumbItem::%s', $itemName)
+        );
+    }
+
+    public function isAccessibleItem(string $itemName)
```
Comment (konandrum) [inline Breadcrumb/Breadcrumb.php]: missing return type :bool

File: Breadcrumb/Breadcrumb.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    {
+        return null !== $this->getTakenPathSource($itemName);
+    }
+
+    private function getTakenPathSource(string $itemName)
```
Comment (konandrum) [inline Breadcrumb/Breadcrumb.php]: missing return type :string

File: Step/Event/Action/GoNextStepEventAction.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            ->setNormalizer('path_index', function (Options $options, $value) {
+                return (int) $value;
+            })
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function doExecute(
```
Comment (konandrum) [inline Step/Event/Action/GoNextStepEventAction.php:41]: single line ?

File: Breadcrumb/Breadcrumb.php [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+{
+    const BACK_QUERY_PARAMETER_ITEM_NAME = '_back_step';
+
+    private $request;
+
+    private $navigator;
+
+    private $items = [];
+
+    public function __construct(Request $request, NavigatorInterface $navigator)
```
Comment (konandrum) [inline Breadcrumb/Breadcrumb.php:25]: Add missing doc string

File: Step/Event/Action/GoNextStepEventAction.php (hunk 4 above)
Comment (BeBlood) [reply Step/Event/Action/GoNextStepEventAction.php:41]: All the other step event actions were done like this, so i should keep this like that ?

**Pull request merged**